charley_tran/week_01/day04_hw.html

```diff
@@ -0,0 +1,152 @@
+<html>
+<script>
+/*Part 1, Rectangle
+
+Given the following a rectangle object like the one below, write the following functions:
+
+isSquare - Returns whether the rectangle is a square or not
+area - Returns the area of the rectangle
+perimeter - Returns the perimeter of the rectangle
+var rectangle = {
+  length: 4,
+  width: 4
+};*/
+
+var rectangle = {
+	length : 4,
+	width : 4
+};
+
+var isSquare = function() {
+	if (rectangle.length = rectangle.width) {
+		return rectangle = 'square'
+	}
+}
+var area = function() {
+	return rectangle.length * rectangle.width
+}
+
+var perimeter = function () {
+	return 2 * (rectangle.length + rectangle.width)
+}
```

Comment:
All is good, just wanted to note that you also can put functions related to rectangle into object rectangle as so:
```js
var rectangle = {
   length: 4,
   width: 4,
   isSquare: function(){
        if(this.length = this.width){
            return "This rectangle is a square";
        }
   }
}